docs/content/release-notes/flink-1.15.md

```diff
@@ -0,0 +1,379 @@
+---
+title: "Release Notes - Flink 1.15"
+---
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+# Release notes - Flink 1.15
+
+These release notes discuss important aspects, such as configuration, behavior, 
+or dependencies, that changed between Flink 1.14 and Flink 1.15. Please read these 
+notes carefully if you are planning to upgrade your Flink version to 1.15.
+
+## Summary of changed dependency names
+
+There are three changes in Flink 1.15 that require updating dependency names when 
+upgrading from earlier versions:
+
+* The newly introduced module `flink-table-planner-loader` (FLINK-25128) could 
+  replace the legacy `flink-table-planner_2.12`. As a consequence, `flink-table-uber` 
+  has been split into `flink-table-api-java-uber`, `flink-table-planner(-loader), 
+  and `table-runtime`. Besides, the artifactId of `flink-sql-client` has no Scala 
+  suffix (_2.11 / _2.12) anymore, and Scala users need to explicitly add a 
+  dependency to `flink-table-api-scala` or `flink-table-api-scala-bridge`.
```

Comment:
This isn't phrased in a user-friendly way. We have to come up with something along the lines of "if you used A, now use B".
IMO we also shouldn't combine so many things into 1 item (split/suffix removal/scala opt-in).

Reply:
Thanks @zentol for the review! I'll split them into different items and give a summary table for the relationship.